hazelcast-openshift-rhel/Dockerfile

```diff
@@ -9,14 +9,15 @@ ENV LANG en_US.utf8
 ENV USER_NAME=hazelcast
 ENV USER_UID=10001
 
-ARG HZ_VERSION=3.8
+ENV HZ_VERSION=3.8
+
 ARG HZ_KUBE_VERSION=1.0.0
 ARG HZ_MAVEN_DIR=${HZ_VERSION}
-ARG REPOSITORY_URL=https://repo1.maven.org/maven2
+ARG REPOSITORY_URL=https://repository-hazelcast-l337.forge.cloudbees.com
 
 LABEL name="hazelcast/hazelcast-openshift-rhel" \
       vendor="Hazelcast, Inc." \
-      version="7.2" \
+      version="${HZ_VERSION}" \
```

Comment:
Seems like I was wrong... can't find the reference I was looking for .... whatever :D